src/api/index.ts

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import axios from "axios";
```

Comment:
We could maybe also remove axios as a dependency, since oazapfts uses the native fetch API internally....